packages/ember-engines/addon/components/link-to-external.hbs

```diff
@@ -0,0 +1,5 @@
+{{~#if (has-block)~}}
+  {{yield}}
+{{~else~}}
+  {{this.linkTitle}}
+{{~/if~}}
```

Comment:
@void-mAlex can we change this syntax? It's not what we're used to seeing in the ember guides

Reply:
it's exactly what https://github.com/emberjs/ember-legacy-built-in-components/blob/095fa3eba7cddf0716ec89fd4bb36087ebe2662f/addon/components/link-to.hbs
the original had which I kept intact
I would advocate for minimal changes with this as while it looks funky all it does is remove whitespace, and that could be a change in behaviour for consumers
so unless you feel really strongly about this I wouldn't touch it